infra/config-store/src/main/java/com/simprints/infra/config/store/local/ConfigLocalDataSourceImpl.kt

```diff
@@ -76,7 +80,21 @@ internal class ConfigLocalDataSourceImpl @Inject constructor(
         configDataStore.updateData { it.toBuilder().clear().build() }
     }
 
-    override suspend fun getDeviceConfiguration(): DeviceConfiguration = deviceConfigDataStore.data.first().toDomain()
+    override suspend fun getDeviceConfiguration(): DeviceConfiguration {
```

Comment:
This works and the overhead in cases where modules are already tokenized (increasing majority) the overhead would probably be super small. Still, it feels wrong to do this check every time instead of just migrate saved modules once to tokenized ones.

Reply:
I agree that a migration would be cleaner, on the other hand, this would ensure 100% consistency in all cases. 
And considering the amount of selected modules is usually in the 1-10 range, I would say the overhead is negligible.